### ISSUE TITLE
Project dead?

### ISSUE THREAD

**Issue by dvc94ch** [odr-group/odr-spec#2]

**Comment by Libermentix** [comment]
@dvc94ch no, it's not dead. We are currently looking for contributors. Would you want to get involved? If yes, let's schedule a call to coordinate and setup a roadmap.

**Comment by dvc94ch** [comment]
I'm currently busy with other projects and don't have the bandwidth to work on this for the foreseeable future.
What is currently missing are the specification of any specific design rules at the very least, otherwise there isn't any real interopability. Idealy there would also be a reference implementation of a drc checker for these design rules and a collection of design rule specifications for the popular osh pcb manufacturers.

**Comment by Libermentix** [comment]
@dvc94ch alright, understood. I will have a look into structuring the roadmap. It's a good point to start by making the obvious todo's more transparent.

**Comment by thesourcerer8** [comment]
I am the developer of altium2kicad, and DRC has been one of the points where I didn't do anything yet, because the DRC capabilities of Altium seemed extremely flexible and mapping those to KiCad seemed very difficult. So perhaps one way to start this is that we try to convert DRC rules from existing tools and real-life projects into odr, and to then try to match them together to see whether this is feasible? 
Another topic I am interested in is Security-Design-Rules and Security-Design-Rule-Checks, which should be able to warn you from open JTAG ports, external access to critical internal i2c busses, ...
